src/navigations/KeyboardWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {ViewStyle, KeyboardAvoidingView, Platform} from 'react-native';
+import dimen from '../utils/dimen';
+
+type Props = {
+  children: any;
+  keyboardVerticalOffset?: number;
+  contentContainerStyle?: ViewStyle;
+};
+
+const KeyboardWrapper = ({children, keyboardVerticalOffset}: Props): JSX.Element => {
+  return (
+    <KeyboardAvoidingView
+      style={{flex: 1}}
+      keyboardVerticalOffset={keyboardVerticalOffset || dimen.normalizeDimen(40)}
+      {...(Platform.OS === 'ios' ? {behavior: 'padding'} : {})}>
+      {children}
+    </KeyboardAvoidingView>
+  );
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `KeyboardWrapper` component does not use the `contentContainerStyle` prop. If this prop is intended to allow customization of the style for the `KeyboardAvoidingView`, it should be applied to the `style` prop of the `KeyboardAvoidingView`.
```diff
    <KeyboardAvoidingView
      style={{flex: 1}}
      keyboardVerticalOffset={keyboardVerticalOffset || dimen.normalizeDimen(40)}
      {...(Platform.OS === 'ios' ? {behavior: 'padding'} : {})}>
+       style={contentContainerStyle || {flex: 1}}
      {children}
    </KeyboardAvoidingView>
  );
```
Please note that you need to destructure `contentContainerStyle` from props at line 12.



<!-- This is an auto-generated comment by OSS CodeRabbit -->